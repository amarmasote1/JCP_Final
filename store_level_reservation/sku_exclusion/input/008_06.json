
[
{"itemID": "80510002012","uom": "EACH", "storeID":"1068"}

]