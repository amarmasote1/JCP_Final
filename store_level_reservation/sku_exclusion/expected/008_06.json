{
	"body": [{
		"status": 200,
		"message": "Success",
		"key": "JCP|1068|80510002012|EACH",
		"body": {
			"invall": {
				
				"shipATP": 40,
				
				"unconfirmeddemand": 0,
				
				
				"ship_active": 1,
				
				
				
				"isDirty": 0,
				"bopusSafetyFactor": 0,
				"bopusATP": 40,
				"shipSafetyFactor": 0,
				"reserveddemand": 0,
				"bopus_active": 1,
				"onhandsupply": 40
			}
		},
		"warnings": null
	}],
	"status": 221,
	"message": "Successful"
}